src/api/libcellml/component.h

```diff
@@ -414,6 +416,16 @@ class LIBCELLML_EXPORT Component: public ComponentEntity, public ImportedEntity
      */
     ComponentPtr clone() const;
 
+    /**
+     * @brief Set the import source of this component.
+     *
+     * If this component is already located in a Model instance, then the
```

Comment:
Should be "... @c Model instance..." Same on the line below.

Reply:
This is inconsistent everywhere ... and should be `@c ModelPtr` or `@c Model` ??

Reply:
My personal preference is for `@c Model`.

Reply:
Will be addressed in #618